Gaijinizer patch/Enemies.txt

```diff
@@ -1679,12 +1677,11 @@ But that's what she thinks.#MTLed
 キスでこの舌を入れられたが最後、相手の人間は
 一転して精力を根こそぎ吸われることになる。
 > CONTEXT: Enemies/563/BookDescription/3
-Her "tongue" is actually the same as the tentacle of slug
-form
-It is very dangerous because of the flow of powerful magic.
-I put this tongue in a kiss, but lastly, the other person
-is
-You will be forced to take energy.#MTLed
+Her "tongue", which functions identically to one
+of the tentacles from her slug form, flows with
+powerful magic and is very dangerous. No sooner
+has she put her tongue in his mouth during a kiss
+has she completely drained his energy.#RonIffyWording
```

Comment:
Let's try rewording that second half a bit (a bit of localization here...):
>_**As soon as
their new companion comes for a kiss, the tables
quickly turn as he is completely
drained of energy.**_

(probably need to debate this later...)